modules/cnf-intel-n3000-optimize-data-plane.adoc

```diff
@@ -0,0 +1,39 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-understanding-intel-hardware-accelertor-card-for-openshift-container-platform_{context}"]
+= Understanding Intel hardware accelerator cards for OpenShift Container Platform
+
+Hardware accelerator cards from Intel accelerate 4G/LTE and 5G Virtualized Radio Access Networks (vRAN) workloads. This in turn increases in turn the overall compute capacity of a commercial off-the-shelf platform.
+
+The hardware accelerator cards detailed in this section are:
+
+* Intel FPGA PAC N3000
+
+* Intel vRAN Dedicated Accelerator ACC100
+
+.Intel FPGA PAC N3000
+
+The Intel FPGA PAC N3000 card needs flashed with the 4G/LTE or 5G bitstreams to support vRAN workloads.
+The Intel FPGA PAC N3000 is a reference FPGA and uses 4G/LTE or 5G Forward Error Correction (FEC) as an example workload accelerating the 5G or 4G/LTE RAN layer 1 (L1) base station network function.
```

Comment:
Please use "forward error correction" instead of the initial caps.  It is not a proper noun.

Please also make this into two sentences or change "accelerating" to "that accelerates".